asv_bench/benchmarks/utils/common.py

```diff
@@ -455,23 +455,30 @@ def trigger_import(*dfs):
     ----------
     *dfs : iterable
         DataFrames to trigger import.
+
+    Returns
+    -------
+    DataFrame or tuple of DataFrames if more than one object was passed
+        Returns exactly the same objects that were passed into the function.
     """
-    assert ASV_USE_STORAGE_FORMAT == "omnisci"
+    if ASV_USE_STORAGE_FORMAT != "omnisci" or ASV_USE_IMPL != "modin":
```

Comment:
to allow usage of `trigger_import` at the file with base benchmarks (`asv_bench/benchmarks/benchmarks.py`)